pkg/management/component.go

```diff
@@ -1450,6 +1453,324 @@ func (c *component) UpdateAuthorizations(ctx context.Context, realmName string,
 	return nil
 }
 
+func (c *component) AddAuthorization(ctx context.Context, realmName string, groupID string, auth api.AuthorizationsRepresentation) error {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+
+	group, err := c.keycloakClient.GetGroup(accessToken, realmName, groupID)
```

Comment:
Romain wrote : "groupID should be optional as there is some rights with Realm scope."
I see his comment as "Resolved" but here, we are getting the group from Keycloak... If groupID is not provided, why do we call Keycloak?

Reply:
This groupID could not be optional, it is the targetGroupID that is optional !